src/test/java/domain_tests/UserTest.java

```diff
@@ -15,41 +15,30 @@
     private Agent nico;
     private Agent jorge;
     private Agent damian;
+    private Agent carmen;
 
     @Before
     public void setUp(){
-        nico = new Agent("Nicolás", "Rivero", "nico@nicomail.com","nico123");
-        jorge = new Agent("Jorge", "Zapatero", "jorge@jorgemail.com", "jorge123", "C/ La calle", "España", "111111111A");
-        damian = new Agent("Damian", "Rubio", "damian@damianmail.com", "damian123", "C/ The street", "Inglaterra", "222222222B");
+        nico = new Agent("Nicolás Rivero", "nico@nicomail.com","nico123");
+        jorge = new Agent("Jorge Zapatero", "jorge@jorgemail.com", "jorge123", "111111111A", 1);
+        damian = new Agent("Damian Rubio", "damian@damianmail.com", "damian123", "222222222B", 2);
+        carmen = new Agent("Carmen", "c@c.com", "cccc", "181818S", 18, 20.48, 156.2);
     }
 
     @Test
-    public void firstNameTest(){
-        Assert.assertEquals("Nicolás", nico.getFirstName());
-        Assert.assertEquals("Jorge", jorge.getFirstName());
-        Assert.assertEquals("Damian", damian.getFirstName());
+    public void nameTest(){
```

Comment:
Usually, IDEs have support for automatic generation of getters and setters. For that reason, and even though it increases coverage, testing these code-paths could be omitted as they don't actually exercise logic that is prone to error. 

In words of Jeff Atwood, [The Best Code is No Code At All](https://blog.codinghorror.com/the-best-code-is-no-code-at-all/)